### DISCUSSION TITLE
请教下，在高可用的部署模式下config-service.cache.enabled这个参数是不是没啥用了，我看目前的缓存实现是guava

### DISCUSSION THREAD

**Discussion by 1123183721** [apolloconfig/apollo#3676] [Q&A] [answered]
请教下，在高可用的部署模式下config-service.cache.enabled这个参数是不是没啥用了，我看目前的缓存实现是guava

**Comment by Anilople** [comment, marked as answer]
这里的没啥用是指？

**Reply by 1123183721** [reply]
默认是不启用的，启用后缓存在jvm内存中，不会有数据不一致的问题么

**Reply by Anilople** [reply]
极端场景下可能会出现数据不一致，但是一般不会

通常开启缓存，有2个需求：

1. 让单台config service可以服务更多的client
2. 数据库宕机时，保障client还可以读到配置

**Reply by 1123183721** [reply]
了解了，感谢
